UPGRADE.md

```diff
@@ -233,7 +233,7 @@ exceptions were reported to clients (e.g. database connection errors, file acces
 Also every error reported to client now has new `category` key which is either `graphql` or `internal`.
 Exceptions implementing `ClientAware` interface may define their own custom categories.
 
-During development or debugging use `$executionResult->toArray(true)`. It will add `debugMessage` key to 
+During development or debugging use `$executionResult->toArray(DebugFlag::INCLUDE_DEBUG_MESSAGE)`. It will add `debugMessage` key to 
```

Comment:
This shouldn't change (as it relates to migration to version 0.10.x. Instead could you add a separate entry in Changelog.md and Upgrade.md about this change?